Project-Timer-frontend/src/app/shared/interfaces/project.interface.ts

```diff
@@ -0,0 +1,8 @@
+import { Task } from "./task.interface";
+
+export interface Project {
+    id: number;
+    projectName: string;
+    secondsClocked: number;
+    tasks: Task[];
+}
```

Comment:
masz lintera ? tutaj github pokazuje brak lintera, plik powinien mieć jeszcze jedną linię czyli nie 8 a 9  :)

Reply:
Eslint and Prettier installed and configured